addon/serializers/github-tree.js

```diff
@@ -0,0 +1,34 @@
+import GithubSerializer from './github';
+
+export default GithubSerializer.extend({
+  normalize(modelClass, resourceHash, prop) {
+    let blobItems = resourceHash.tree
+      .filter(item => item.type === 'blob');
+    let treeItems = resourceHash.tree
+      .filter(item => item.type === 'tree');
+
+    let normalizedHash = {
+      id: resourceHash.sha,
+      sha: resourceHash.sha,
+      url: resourceHash.url,
+      files: blobItems
+        .reduce((files, blob) => {
```

Comment:
Would prefer this moved up onto the same line now that it's just a variable name, better readability (and below as well for the other 3 calls)